Source/Details/_ASDisplayView.h

```diff
@@ -20,8 +20,16 @@
 // This class is only for use by ASDisplayNode and should never be subclassed or used directly.
 // Note that the "node" property is added to UIView directly via a category in ASDisplayNode.
 
+@class ASDisplayNode;
+
 @interface _ASDisplayView : UIView
 
+/**
+ @discussion This property overrides the CALayer category method which implements this via associated objects.
+ This should result in much better performance for _ASDisplayLayers.
+ */
+@property (nonatomic, weak) ASDisplayNode *asyncdisplaykit_node;
```

Comment:
We may should add proper nullable annotations for this and in `_ASDisplayLayer` too as both could be nil.